drivers/couchbase/couchbase-driver/src/main/java/io/mongock/driver/couchbase/repository/CouchbaseRepositoryBase.java

```diff
@@ -35,8 +35,8 @@
 public class CouchbaseRepositoryBase <DOMAIN_CLASS> implements EntityRepository<DOMAIN_CLASS, JsonObject>, RepositoryIndexable {
   private final static Logger logger = LoggerFactory.getLogger(CouchbaseRepositoryBase.class);
   private final static String INDEX_NAME = "idx_mongock_keys";
-  protected final static String DOCUMENT_TYPE = "mongock";
-  protected final static String DOCUMENT_TYPE_KEY = "_type";
+  protected final static String DOCUMENT_TYPE_CHANGE_ENTRY = "mongockChangeEntry";
```

Comment:
This ChangeEntry specific code should be moved to `CouchbaseChangeEntryRepository`.
Additionally, for LockEntry entity we should add another document type, named "mongockLockEntry", and assign it in its repository `CouchbaseLockRepository`.